packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -601,7 +601,8 @@ class _AnimatedPositionedState extends AnimatedWidgetBaseState<AnimatedPositione
 /// Animated version of [Opacity] which automatically transitions the child's
 /// opacity over a given duration whenever the given opacity changes.
 ///
-/// Animating an opacity is relatively expensive.
+/// Animating an opacity is relatively expensive because it requires repeatedly
+/// painting the child into an intermediate buffer every frame of the animation.
```

Comment:
Why is that?  It should be a composited opacity effect, which means we just re-composite with the new opacity value.  (All Opacity widgets are composited in our system.)